CSharp/TextSearch/JobSubmitter/JobSubmitter.cs

```diff
@@ -94,7 +94,7 @@ public JobSubmitter()
                 //Define the pool specification for the pool which the job will run on.
                 PoolSpecification poolSpecification = new PoolSpecification()
                     {
-                        TargetDedicated = numberOfPoolComputeNodes,
+                        TargetDedicatedComputeNodes = numberOfPoolComputeNodes,
                         VirtualMachineSize = "small",
                         //You can learn more about os families and versions at: 
                         //http://azure.microsoft.com/documentation/articles/cloud-services-guestos-update-matrix
```

Comment:
Aargh, I'm an idiot.  For some reason I misread this as "VM sizes" because why would you not.  Sorry, ignore me!